src/pypilecore/input/grouper_properties.py

```diff
@@ -184,11 +193,16 @@ def create_grouper_payload(
     payload["cpt_objects"] = cpt_objects
 
     # validate pile tip levels
-    raw_lengths = [frozenset(values) for values in pile_tip_level_object.values()]
+    raw_lengths = [
+        frozenset(np.round(values, 1)) for values in pile_tip_level_object.values()
+    ]
     if len(list(set(raw_lengths))) > 1:
         msg = "For the grouper payload must all CPT's have a valid bearing capacity for all pile tip levels. \n"
```

Comment:
Deze zin loopt niet lekker, misschien: "The PileCore grouper requires all CPTs to have a valid bearing capacity for all pile tip levels"